chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -605,6 +605,15 @@ def miimon=(millisecs)
       self
     end
 
+    def xmit_hash_policy
+      sysfs("bonding/xmit_hash_policy")
```

Comment:
Are we sure that this is always present? It seems setting ```xmit_hash_policy``` does only make sense for certain bonding modes (2, 4 and 5 according to https://www.kernel.org/doc/Documentation/networking/bonding.txt). Is the file also present when the bond is currently using any of the other modes?